<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Button_Result">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Посчитать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1stElement">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2ndElement">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_result">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_plus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_minus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_multiply">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_divine">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sqrt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>√</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1stElement">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Первое слагаемое</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2ndElement">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Второе слагаемое</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cos">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_arcsin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>arcsin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_arccos">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>arccos</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
